architecture.puml

```diff
@@ -0,0 +1,18 @@
+@startuml Image Scanner architecture
```

Comment:
Should we move this file into a docs folder?

Reply:
Sounds like a good idea, fixed now.